config/settings.yml

```diff
@@ -886,10 +886,6 @@
     :default: 1.minutes
     :host: 20.minutes
     :vm: 20.minutes
-  :concurrent_requests:
-    :historical: 1
-    :hourly: 1
-    :realtime: 20
```

Comment:
I think we need a data migration for this to remove any existing change records in the database.

Reply:
Or perhaps move from "main" to "ems_vmware"

Reply:
Thanks - so we _should_ have a data migration for any existing settings.